src/coreclr/vm/ClrEtwAll.man

```diff
@@ -1742,6 +1742,21 @@
                         </UserData>
                     </template>
 
+                    <template tid="ContentionStart_V2">
```

Comment:
I wonder if putting the LockId and OwnerId wouldn't be better on the Stop event, WDYT?

Reply:
I think it makes more sense on the start event, since you want to know what was holding it that triggered the contention